assets/js/app.js

```diff
@@ -80,6 +80,29 @@ $(function() {
   })
 });
 
+// download page os buttons
+
+$(function() {
+  const contentAvailable = "Fork the O3DE source and build the engine and tools directly from any branch. For more details, read the documentation.";
+  const contentUnavailable = "Coming soon.";
+  $(".download-os-button").click(function(event){
+    event.preventDefault()
+    
+    $(".download-os").removeClass("active");
+    $(this).parent().addClass("active");
+
+    if($(this).text().includes("Windows")){
+      $(".download-os-content").text(contentAvailable);
+      $("#download-page-buttons a").removeClass("disabled");
+      $("#download-page-buttons a").attr('href', 'https://github.com/o3de/o3de');
+    } else {
+      $(".download-os-content").text(contentUnavailable);
+      $("#download-page-buttons a").addClass("disabled");
```

Comment:
I don't think we should say that you can't or shouldn't fork from GitHub - there is "experimental" support for macOS and "in-progress" (maybe even considered as stable as Windows?) for Linux. Even if there's not an installer available users should be encouraged to fork.